convert_test.go

```diff
@@ -93,9 +93,13 @@ func Test_ToString(t *testing.T) {
 		{time.Date(2000, 1, 1, 12, 34, 56, 0, time.UTC), "2000-01-01 12:34:56"},
 		{struct{ Name string }{"John"}, "{John}"},
 		{[]string{"Hello", "World"}, "[Hello World]"},
+		{[]int{42, 21}, "[42 21]"},
+		{[2]int{42, 21}, "[42 21]"},
+		{[]interface{}{[]int{42, 21}, 42, "Hello", true, []string{"Hello", "World"}}, "[[42 21] 42 Hello true [Hello World]]"},
```

Comment:
can you add [][]int testcase

Reply:
https://github.com/gofiber/utils/pull/63/commits/fb9aac5b7b2b1acb8146ae3445fcd33f811f3a04